hibernate-community-dialects/src/main/java/org/hibernate/community/dialect/AltibaseDialect.java

```diff
@@ -500,11 +500,6 @@ public boolean qualifyIndexName() {
 		return false;
 	}
 
-	@Override
-	public boolean supportsInsertReturningGeneratedKeys() {
```

Comment:
Removed this because Statement.getGeneratedKeys() is not fully supported by Altibase.